index.js

```diff
@@ -125,7 +123,11 @@ const preprocessor = (options = {}) => {
 
       if (stats.hasErrors()) {
         err = new Error('Webpack Compilation Error')
-        err.stack = jsonStats.errors.join('\n\n')
+        err.message += `\n${jsonStats.errors
+        // remove stack trace lines since they're useless for debugging
+        .map((err) => err.replace(/\n\s*at.*/g, '').replace(/From previous event:\n?/g, ''))
```

Comment:
this same logic is used in test runner to strip stack trace lines, so we're just moving this logic up
https://github.com/cypress-io/cypress/blob/master/packages/server/lib/plugins/preprocessor.coffee#L16

Reply:
ughh, this is so so weird - why have 3 lines of code inside a string template?

Reply:
eslint auto-fixed to that when i tried to add `'\n'`

Reply:
fixed